packages/aic/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import * as crypto from '@adxp/crypto'
+import { database, Database } from './db'
+import { server } from './server'
+
+const run = async () => {
+  // create or open db
+  const dbLoc = 'adx.sqlite'
```

Comment:
we may want this to be more configurable (similar to the PDS) where I can pass in which ever database instance I like - an in-memory DB for instance

Reply:
thanks, though I had put that back to an ENV var